src/Controller.html

```diff
@@ -8,6 +8,13 @@
       ref:progress on:click="handleProgressClick(event)"
     >
       <div class="rr-progress__step" ref:step style="width: {percentage}"></div>
+        {#each getCustomEvents as event,i (i)}
+          <div
+            title="{event.name}"
+            style="width: 10px;height: 5px;position: absolute;top: 2px;transform: translate(-50%, -50%);background: {event.background};left: {event.position};"
```

Comment:
how about move the static styles to CSS?

Reply:
Actually I have tried a lot on moving this part to the css class and I have constantly observed strange behavior like whenever I move this to class i.e: `.tag` the final build does not includes the css. It breaks the component.

FYI: I am not much familiar with Svelte.

Reply:
It's okay, I will give a try after merge